iterations_runners/iterations_runner.java

```diff
@@ -19,13 +19,15 @@
     public static void main(String args[]) throws
         ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, java.lang.reflect.InvocationTargetException {
 
-        if (args.length != 3) { // not 4 since java doesn't put the program name in args[0]
-            System.out.println("usage: iterations_runner <benchmark> <# of iterations> <benchmark param>\n");
+        if (args.length != 4) { // not 4 since java doesn't put the program name in args[0]
```

Comment:
I don't think the comment here is very useful (comparing Java to other languages use of args is a bit confusing IMHO).